### ISSUE TITLE
Cannot create MySQL connections

### ISSUE THREAD

**Issue by ltning** [rickbergfalk/sqlpad#486]
When trying to create a connection (mysql) I'm taken straight back to the front page and no connection is created. If I try the same with "mock" driver it works. I'm running with --debug and I don't see any POSTs when creating the connection (as opposed to when creating one with mock driver):
```
GET /api/app 200 2.820 ms - 230
POST /api/signup 200 523.148 ms - 2
GET /api/app 200 1.033 ms - 322
POST /api/signin 200 247.889 ms - 2
GET /api/app 200 0.918 ms - 322
GET /api/queries 200 0.996 ms - 14
GET /api/connections/ 200 3.037 ms - 18
GET /api/tags 200 3.763 ms - 11
GET /queries/api/app 200 2.188 ms - 322
GET /api/drivers 200 0.822 ms - 5049
GET /queries/new?name=3ds_mdadb39&driver=mysql&host=db02.staging.modirum.com&port=3306&database=3ds_mdadb39&username=sqlpad&password=d6LJ8nzFY3I3BKkR 200 0.959 ms - 2182
GET /queries/api/app 200 0.845 ms - 322
GET /api/queries 200 1.011 ms - 14
```

**Comment by rickbergfalk** [comment]
This one is a bit odd - I wasn't able to reproduce at all. Is something running in front of SQLPad? 

That GET request appears as if it has all the config values that should have been sent via POST

**Comment by ltning** [comment]
There is an nginx in front, but as I mentioned it worked with the Mock driver. When using that, I get the appropriate POST request as well. When bypassing the nginx proxy I get exactly the same behaviour. I tried debugging in Firefox but that is not my strong suit so I got nowhere.

**Comment by ltning** [comment]
Btw I cannot add users either, much the same behaviour:

```
GET /api/users 200 1.932 ms - 198
GET /queries/new?email=ltning-sqlpad-staging%40xyz.net&role=admin 304 1.141 ms - -
GET /queries/api/app 200 0.990 ms - 322
GET /api/queries 200 1.031 ms - 14
GET /api/connections/ 200 2.137 ms - 1495
```

If I whitelist the domain I am able to sign up - but I cannot do that as it's a public instance. I see no POST event in the webserver log either, nor do I see any errors there. Again same behaviour if I'm talking directly to the sqlpad instance instead of via nginx.

**Comment by rickbergfalk** [comment]
@ltning This should be corrected in 3.5.2. Let me know if that is not the case and we can reopen issue
